Sources/LaunchAtLogin/copy-helper-swiftpm.sh

```diff
@@ -1,19 +1,29 @@
 #!/bin/bash
 
+verlte() {
+    [  "$1" = "`echo -e "$1\n$2" | sort -V | head -n1`" ]
+}
+
+if verlte "10.14.4" "$MACOSX_DEPLOYMENT_TARGET"; then
+    helper_name="LaunchAtLoginHelper"
+    checksum="4842fa690b83d96d509dad6763571125e4a2ae2dcf8f858f88518486816a5598"
```

Comment:
Make the checksums top-level constants to make it easier to update them in the future.

Reply:
May you please clarify what you mean `top-level`? Something like this?

```
#!/bin/bash

HELPER_CHECKSUM="4842fa690b83d96d509dad6763571125e4a2ae2dcf8f858f88518486816a5598"
HELPER_CHECKSUM_RUNTIME="ceef772a05157e9b64e40fb022d19e6462e371f9d69d1dbd3c1e64096bde535d"
...
checksum="$HELPER_CHECKSUM"
```

Reply:
Correct